core/templates/dev/head/pages/collection_editor/editor_tab/CollectionLinearizerService.js

```diff
@@ -205,12 +208,15 @@ oppia.factory('CollectionLinearizerService', [
           return false;
         }
 
+        var linearNodeList = _getCollectionNodesInPlayableOrder(collection);
```

Comment:
it's so that the skill associated to the node can be deleted as well (originally only the node is deleted)